crates/cairo-lang-syntax-codegen/src/cairo_spec.rs

```diff
@@ -177,7 +177,7 @@ pub fn get_spec() -> Vec<Node> {
         .node("rbrack", "TerminalRBrack")
     )
     .add_struct(StructBuilder::new("ExprInlineMacro")
-        .node("name", "TerminalIdentifier")
+        .node("path", "ExprPath")
```

Comment:
I think we actually want a `path` here so macros can be imported similar to other modules?